src/test/java/com/fakng/fakngagrgtr/parser/AppleParserTest.java

```diff
@@ -70,8 +64,12 @@ public void testParseCreatesVacanciesFromJsonWithCachedLocations() throws IOExce
                 .thenReturn(company.getLocations().get(0));
         Mockito.when(locationProcessor.processLocation(company, "city_2", "country_2"))
                 .thenReturn(company.getLocations().get(1));
-        Mockito.when(htmlWebClient.getPage(URL))
-                .thenReturn(new WebClient().loadHtmlCodeIntoCurrentWindow(Files.readString(Paths.get("src/test/resources/com/fakng/fakngagrgtr/parser/apple-data.html"))));
+        try (WebClient trueClient = new WebClient()) {
+            Mockito.when(htmlWebClient.getPage(URL))
+                    .thenReturn(trueClient.loadHtmlCodeIntoCurrentWindow(
+                            Files.readString(Paths.get("src/test/resources/com/fakng/fakngagrgtr/parser/apple-data.html"))
```

Comment:
I forgot to create a constant variable for the path. Do you think it should be done? Something like PATH_TO_TEST_DATA

Reply:
Well, makes sense